integrations/woopra/HISTORY.md

```diff
@@ -1,4 +1,12 @@
 
+
+2.1.0 / 2022-02-18
```

Comment:
I think since we're no longer passing through some options to `woopra` this should be considered a major update.

Reply:
That's up to Segment to decide. We're happy to bump this to 3.0.